### DISCUSSION TITLE
Welcome to Natural Discussions!

### DISCUSSION THREAD

**Discussion by rlogiacco** [rlogiacco/Natural#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by athos-georgiou** [comment]
Hey, 

It looks like in Eclipse Marketplace your project points to is: http://github.com/rlogiacco/Natural and it's not possible to install via said Marketplace. Is this something you would would be able to correct?

Thanks

**Reply by iamrizwan007** [reply]
The URL has been updated from **https://rlogiacco.github.com/Natural** to **https://rlogiacco.github.io/Natural**
The same has not been updated in the marketplace and the github repo.
you can use the updated url **https://rlogiacco.github.io/Natural** and install from eclipse following these steps:
1. click on help
2. select install new softwares
3. enter the given URL in  Work with field and click add
4. name it as "Natural" on the pop up and click on add
5. follow the process of next, next, accept terms and condition and finish.

you are all set to use Natural plugin. Hope this helps. Thanks

**Reply by vaninair2016** [reply]
Hi , I am unable to get this added from Help in eclipse . It is showing below error. Can you please help @rlogiacco 


Unable to read repository at https://rlogiacco.github.io/Natural/content.xml.
Unable to read repository at https://rlogiacco.github.io/Natural/content.xml.
PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target



thanks,
Vani

**Comment by ashwinstrikes** [comment]
Natural plugin not available in eclipse marketplace, pls assist

**Comment by devisuresh** [comment]
Please unsubscribe me from your mailing list

**Comment by rlogiacco** [comment]
There’s an unsubscribe link as footer of this email: use that.
You subscribed, you unsubscribe.

Il giorno sab 14 gen 2023 alle 23:56 devisuresh ***@***.***>
ha scritto:

> Please unsubscribe me from your mailing list
>

**Comment by rlogiacco** [comment]
Hi Vininair, the plug-in has been having this issue from quite a long time,
but I haven’t found the time to fix it. If you have a pull request that
fixes it, I’m willing to accept it.

I don’t know if any fork of this repo has already fixed it.

There are some workaround that have been discussed her, I don’t know which
one works.

I suggest scavenging the issues for a resolution or seek help from the
community.

Regards,
   Roberto

Il giorno sab 15 lug 2023 alle 06:22 vaninair2016 ***@***.***>
ha scritto:
